src/components/AppIndex.vue

```diff
@@ -52,6 +76,36 @@ export default {
             })
             return deferred.promise()
         },
+        /**
+         * Check if recipe should be displayed, depending on selected keyword filter.
+         * Returns true if recipe contains all selected keywords.
+         */
+        recipeVisible: function(keywords) {     
+            if (this.keywordFilter.length == 0) {
+                return true;
+            } else {
+                if (!keywords) return false;
+                let kw_array = keywords.split(',');
+                return this.keywordFilter.every(kw => kw_array.includes(kw));
+            }
+        },
+        /**
+         * Extract and set list of keywords from the returned recipes
+         */
+        setKeywords: function(recipes) {
+            this.keywords = []
+            if ((recipes.length) > 0) {
+                recipes.forEach(recipe => {
+                    if(recipe['keywords']) {                    
+                        recipe['keywords'].split(',').forEach(kw => {
+                            if(!this.keywords.includes(kw)) {
+                                this.keywords.push(kw)                           
+                            }
+                        })
+                    }                    
+                })
```

Comment:
We might need to introduce a scroll view at some point, but for now this should be fine 

<img width="971" alt="Screenshot 2020-11-08 at 22 35 08" src="https://user-images.githubusercontent.com/7623143/98484896-e2842980-2212-11eb-8eec-45d37a68ea90.png">

Reply:
That will depent how many recipes you have in your cookbook. Some pages set a certain amount of keywords to the tag per recipe. Of course, there will be similarities but for a few 100 recipes the amount of keywords might become significant. But you are right, this might be ok if done later.